spec/FrameSpec.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+describe("Frame", function() {
+  var frame;
+
+  beforeEach(function() {
+    frame = new Frame();
+  });
+
+  describe("Calculates scores", function() {
+    it("can put the score after a roll into the roll array", function() {
+      frame.roll(7);
+      expect(frame._rolls).toEqual([7]);
```

Comment:
Try to avoid testing state.  Test behaviour instead.  Is there a public behaviour that you can test that happens because 7 has been rolled?